R/mod_datatable_filters.R

```diff
@@ -1,3 +1,17 @@
+#' Filtering operator
+#'
+#' @description A special operator that doesn't filter nulls
+#' @rdname special_pipe
+#' @export
+
+`%==%` <- function (e1, e2) {
```

Comment:
Can we name this something that hints at NULL? Maybe %!NULL%, %NULL%, %null%. The `==` makes me think of comparing two values for equality.